Web/starter_project_iconics/pages/profile/index.tsx

```diff
@@ -0,0 +1,35 @@
+import BlogCard from '@/components/common/blogCard'
+import { card1 } from '@/store/mockData/data'
+
+const index = () => {
+  const nums: number[] = [0, 1, 2, 3, 4]
```

Comment:
there might be a better way to loop 4 times other than creating a list of numbers
reading down, it looked to me like it had another purpose that i didn't see

at least I would recommend making a list of what kind of data would be populated in the card (like [{card: card1, feature: "likes"}]) and duplicating them to have four items 

at least that way I know what that list represents (and the naming of it should also explain that)

Reply:
or if the data is coming from mock anyways, it might be better to have that list of duplicates be created in the mocks just the way you would use it